src/frontEnd/reactChecks/createSummary.js

```diff
@@ -33,9 +41,44 @@ const securitySummary = (summary, reports) => {
     description: 'Existe un .env con variables de entorno en el proyecto',
     value: reports.some(elem => elem.metric === envMetrics.ENV_IS_USED && elem.value)
   });
+  summary.push({
+    metric: 'SUMMARY-SECURITY-2',
+    description:
+      'Las credenciales para firmar a producción se encuentran en un lugar seguro y disponible para el equipo de desarrollo y el TM',
+    value: 'Manual'
+  });
+  summary.push({
+    metric: 'SUMMARY-SECURITY-3',
+    description: 'No se guardan claves secretas en texto plano o en constantes dentro de la aplicación',
+    value: 'Manual'
+  });
+  summary.push({
+    metric: 'SUMMARY-SECURITY-5',
+    description: 'Las contraseñas no son nunca visibles para el usuario despues del login',
+    value: 'Manual'
+  });
 };
 
 const buildingSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-BUILDING-1',
+    description: 'El proyecto utiliza la ultima o anteultima versión del framework',
+    value: !reports.some(
+      elem => elem.metric === generalMetrics.OUTDATED_DEPENDENCIES && elem.value.includes('react')
+    )
+  });
+  summary.push({
+    metric: 'SUMMARY-BUILDING-2',
+    description: 'Las dependencias del proyecto están actualizadas',
```

Comment:
Should this wording be changed to something like 'No se superan las {maxUnusedDependencies} dependencias  sin usar' ?